static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js

```diff
@@ -39,44 +45,47 @@ export const acquisitionsBannerFivTemplate = (
     <div class="fiv-banner__container-${
         isPotential ? `potential` : `existing`
     }">
-        <div class="fiv-banner__text">
         ${
             isPotential
```

Comment:
with more time and if this was going in longer term than just a week I'd be tempted to store the text in a data structure and map it into this html format, but at the moment it looks a bit hacky.